FLEDGE.md

```diff
@@ -501,7 +499,7 @@ The response from the server should be a JSON object of the form:
 }
 ```
 
-and the server must include the HTTP response header `X-fledge-bidding-signals-format-version: 2`.  If the server does not include the header, the response will assumed to be an in older format, where the response is only the contents of the `keys` dictionary.
+and the server must include the HTTP response header `Ad-Auction-Bidding-Signals-Format-Version: 2`.  If the server does not include the header, the response will assumed to be an in older format, where the response is only the contents of the `keys` dictionary.
```

Comment:
I have an approved [PR](https://github.com/WICG/turtledove/pull/772/files#diff-d65ba9778fe3af46de3edfce2266b5b035192f8869280ec07179963b81f4e624) which changes the headers in both the explainer and the spec. And under that PR, Matt [suggested](https://github.com/WICG/turtledove/pull/772#issuecomment-1698147560) not update this header in explainer and spec to not advertise the new name for a header we're going to remove.

Reply:
Since we never supported the new header name, I reverted this line.